### PR TITLE
fix(#6481,#6489) feedback attempt #2

### PR DESCRIPTION
**Pull request by ovari**
https://github.com/moneymanagerex/moneymanagerex/issues/6481

https://github.com/moneymanagerex/moneymanagerex/issues/6489#issuecomment-1918085707

Please do not forget to update the mmex.pot file and write information about the fixed bug in the prerelease [page](https://github.com/moneymanagerex/moneymanagerex/releases).

- - -
This change is [Reviewable](https://reviewable.io/reviews/moneymanagerex/moneymanagerex/6495)

### CODE REVIEW

**Review by n-stein — COMMENTED** [review]
Maybe we should leave the `(*.mmb)` and `(*.emb)` part alone. Just wrap `_(MMEX Database)` and `_(Encrypted MMEX Database)` for translation?

**Comment by ovari** [comment]
> Maybe we should leave the `(*.mmb)` and `(*.emb)` part alone. Just wrap `_(MMEX Database)` and `_(Encrypted MMEX Database)` for translation?

Done

**Comment by ovari** [comment]
@n-stein should the strings shown at `https://crowdin.com/editor/moneymanagerex/470/en-hu?view=comfortable&filter=basic&value=0#q=(*.` be changed too?

File: src/mmframe.cpp [hunk 1]
```diff
@@ -2245,7 +2245,7 @@ void mmGUIFrame::OnNew(wxCommandEvent& /*event*/)
         _("Choose database file to create"),
         wxEmptyString,
         wxEmptyString,
-        _("MMB Database (*.mmb)|*.mmb"),
+        _("MMEX Database")+" (*.mmb)|*.mmb"),
```
Comment (n-stein) [inline src/mmframe.cpp:2248]: There is an extra `)` in this line which is causing the build to fail.

**Comment by n-stein** [comment]
> should the strings shown at https://crowdin.com/editor/moneymanagerex/470/en-hu?view=comfortable&filter=basic&value=0#q=(*. be changed too?

I would leave them as-is

**Pull request merged**